sources/logdog/src/log_request.rs

```diff
@@ -205,12 +218,114 @@ where
     Ok(())
 }
 
+/// Copies all file matching the glob pattern given by `request.instructions` to the tempdir with filename and path
+/// same as source file.
+fn handle_glob_request<P>(request: &LogRequest<'_>, tempdir: P) -> Result<()>
+where
+    P: AsRef<Path>,
+{
+    let glob_paths = glob(request.instructions).context(error::ParseGlobPattern {
+        pattern: request.instructions.to_string(),
```

Comment:
```suggestion
        pattern: request.instructions,
```
Shouldn't need to convert `&str` to `String`, snafu does that for you, and it only converts if it's needed during an actual error situation.